### PR TITLE
[11.0][MIG] account_analytic_default migration

### PR DESCRIPTION
**Pull request by yuriqp**
Depends on:

- [x] sale - #1096 
- [x] stock
- [x] sale_stock - #1196 

Description of the issue/feature this PR addresses:
account_analytic_default migration

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]
OK, but we can't merge until the dependencies are merged. I have put them on the PR summary

**Comment by jholze** [comment]
@pedrobaeza Hey Pedro, i need a feedback because of the stock module. Its still a blocker and i don't know if you already worked on it or not. We could do the whole migration of stock, but we need your ok. Thanks Pedro

**Pull request merged**
